e2e/test/provisioning/ProvisioningCertificateValidationE2ETest.cs

```diff
@@ -34,73 +37,113 @@ public static void TestClassSetup(TestContext _)
         [Timeout(TestTimeoutMilliseconds)]
         public async Task ProvisioningServiceClient_QueryInvalidServiceCertificateHttp_Fails()
         {
-            using var provisioningServiceClient = new ProvisioningServiceClient(
-                TestConfiguration.Provisioning.ConnectionStringInvalidServiceCertificate);
+            // arrange
+            using var provisioningServiceClient =
+                new ProvisioningServiceClient(
+                    TestConfiguration.Provisioning.ConnectionStringInvalidServiceCertificate,
+                    new ProvisioningServiceClientOptions()
+                    {
+                        RetryPolicy = new ProvisioningServiceNoRetry(),
+                    });
```

Comment:
```suggestion
            using var provisioningServiceClient = new ProvisioningServiceClient(
                TestConfiguration.Provisioning.ConnectionStringInvalidServiceCertificate,
                new ProvisioningServiceClientOptions
                {
                    RetryPolicy = new ProvisioningServiceNoRetry(),
                });
```

Reply:
No reason to make newlines quite so aggressively. Plenty of line length remaining for the initial call to new.